aws/resources/templates/invitation_template.html

```diff
@@ -1,12 +1,80 @@
-<!doctype html>
-<html>
-    <head>
-        <meta charset="utf-8">
-        <title>Wedding Invitation for the Flying Js!</title>
-        <meta name="description" content="Invitation to the wedding of Jenny Rosenberger and Jesse Kelly.">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    </head>
-    <body>
-        <p>You're invited!</p>
-    </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Invitation | Flying J Wedding</title>
+  <link rel="icon" 
+      type="image/png" 
+      href="logo-favicon.png">
+
+  <!-- CSS  -->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  
+<link href="https://fonts.googleapis.com/css?family=Cormorant" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <link href="../public/css/template-style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
```

Comment:
For local development, it's OK for you to use `../public/css/template-style.css`, but before creating a pull request change to `css/template-style.css`